packages/rfw/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
+
+* Add more testing to restore coverage to 100%.
```

Comment:
Nit: Adds

(Per repo style guide.)

Reply:
Alternately you could omit the CHANGELOG since it's a dev-only change (modulo the production code removal, but you're not mentioning that part anyway).

Reply:
fixed